### DISCUSSION TITLE
Release date for Silicon Labs Arduino Core update

### DISCUSSION THREAD

**Discussion by lboue** [SiliconLabs/arduino#33] [Q&A] [answered]
Hello,

Could you tell us when will the Silicon Labs Arduino Core new version be published?

> The new version is planned to be released around the 21st of March during [Arduino Days](https://days.arduino.cc/about) to support the new Nano Matter board.

I am waiting support for Matter generic switch device: [[Feature request] Momentary switch device example #11]( https://github.com/SiliconLabs/arduino/issues/11#issuecomment-1946036803)

Regards

**Comment by silabs-bozont** [comment, marked as answer]
Hello lboue,
The new [Nano Matter](https://store.arduino.cc/pages/nano-matter) board just launched during Arduino Days, it's in a waitlist order phase as of now - and will start to ship around the end of April.
The original plan for the release was Arduino Days, but we agreed that we'll postpone releasing the updated core till the pre-order boards start shipping.
I'm sorry for this delay and for keeping you waiting more. I hope this extra time will fly by quickly and you'll be able to program your Matter Switch soon.

**Reply by lboue** [reply]
Thanks for the reply. I will continue to use my Silicon labs board with Simplicity studio.

**Reply by lboue** [reply]
Shipping is coming 
![image](https://github.com/SiliconLabs/arduino/assets/938089/1f3e124e-42d4-4c8b-a6d7-071229286393)

**Comment by lboue** [comment]
Version 2.0.0 was released today.
